public/app/core/components/AppChrome/TopBar/SingleTopBar.tsx

```diff
@@ -115,11 +116,20 @@ const getStyles = (theme: GrafanaTheme2, menuDockedAndOpen: boolean) => ({
     justifyContent: 'space-between',
 
     [theme.breakpoints.up('lg')]: {
-      gridTemplateColumns: '2fr minmax(440px, 1fr)',
+      gridTemplateColumns: '2fr minmax(50%, 1fr)',
       display: 'grid',
-
       justifyContent: 'flex-start',
+      ...(menuDockedAndOpen && {
+        gridTemplateColumns: '2fr minmax(55%, 1fr)',
+      }),
+    },
+    [theme.breakpoints.up('xl')]: {
+      gridTemplateColumns: '2fr minmax(40%, 1fr)',
+      ...(menuDockedAndOpen && {
+        gridTemplateColumns: '2fr minmax(55%, 1fr)',
+      }),
     },
+    [theme.breakpoints.up('xxl')]: menuDockedAndOpen && { gridTemplateColumns: '2fr minmax(40%, 1fr)' },
```

Comment:
i don't think we should have this many different breakpoint styles... is this just a side effect of using a percentage in the first case? 🤔

Reply:
Yeah, I reverted the changes and just increased a bit the minimum. If needed, we can always iterate on this in the future.